spec/npm/npmSpec.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+
+describe('The npm module', function() {
```

Comment:
Let's keep this simple, in line with the other describe names.

```suggestion
describe('npm', function() {
```